### PR TITLE
Add examples to the site

### PR DESCRIPTION
**Pull request by dualBreath**
Task: https://github.com/networkservicemesh/deployments-k8s/issues/10487

Add links to examples to the site

### CODE REVIEW

**Comment by dualBreath** [comment]
Header:
Screenshot 2023-11-28 at 14 48 39

Body:
Screenshot 2023-11-28 at 14 48 28

**Comment by dualBreath** [comment]
@denis-tingaikin, @glazychev-art, @NikitaSkrynnik, @d-uzlov, @VitalyGushin please take a look

File: content/docs/concepts/examples.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+title = "Examples"
+description = "NSM use-case examples"
+weight = 6
+date = "2023-11-28"
++++
+
+## Examples
+A number of prepared examples of different ways to use nsm
+
+* [Basic examples](https://github.com/networkservicemesh/deployments-k8s/tree/release/v1.11.0/examples/basic)
```
Comment (d-uzlov) [inline content/docs/concepts/examples.md:11]: I think we should use latest release in links: `v1.11.1`

Comment (dualBreath) [reply content/docs/concepts/examples.md]: Thanks, fixed

**Comment by dualBreath** [comment]
Moved to `Setup` section:
Screenshot 2023-11-29 at 14 09 44

**Pull request merged**